config/accounts_test.go

```diff
@@ -0,0 +1,114 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccountGDPREnabledForRequestType(t *testing.T) {
+	tests := []struct {
+		description          string
+		giveRequestType      RequestType
+		giveGDPREnabled      *bool
+		giveAMPGDPREnabled   *bool
+		giveAppGDPREnabled   *bool
+		giveVideoGDPREnabled *bool
+		giveWebGDPREnabled   *bool
+		wantEnabled          *bool
+	}{
+		{
+			description:        "GDPR AMP integration enabled, general GDPR disabled",
+			giveRequestType:    RequestTypeAMP,
+			giveGDPREnabled:    &[]bool{false}[0],
```

Comment:
So you are setting `giveGDPREnabled` to the address (`&`) of the first element (`[0]`) of a slice of booleans (`[]bool`) with just one element, set to false (`{false}`)? Would it be simpler to set `bfalse := false` and then `&bfalse` here? And likewise for true values?

Reply:
👍 @laurb9 raised a similar point. I did consider that but chose this so everything was in line but I understand it's a bit more complicated. I'm more than happy to change it to what you're both suggesting as the consensus appears to favor this approach.